cli/bench_command.go

```diff
@@ -69,38 +69,73 @@ const (
 
 func configureBenchCommand(app commandHost) {
 	c := &benchCmd{}
+
+	benchHelp := `
+Core NATS publish and subscribe:
+
+nats bench benchsubject --pub 1 --sub 10
```

Comment:
Maybe indent the example commands by 2 or 3 chars so they stand out from the text? Assuming the help parser parses that right

Reply:
Added a couple of spaces, good idea and parser ok with it